### PR TITLE
Add links to learning guides for the tech stack. #221

### PR DESCRIPTION
**Pull request by dabreegster**
Pete is the newest to go through this process, so you may have opinions here!

### CODE REVIEW

File: docs/developer_guide.md [hunk 1]
```diff
@@ -14,6 +14,22 @@ There are some related repositorites also part of ATIP:
...
+- Svelte
+  - [Tutorial](https://svelte.dev/tutorial/basics)
+  - [Examples](https://svelte.dev/examples/hello-world)
+  - [Deep dive on reactivity](https://blog.thoughtspile.tech/2023/04/22/svelte-state/)
+- MapLibre
+  - [Examples](https://maplibre.org/maplibre-gl-js-docs/example/)
+  - Join the `#maplibre` channel on [osmus Slack](https://slack.openstreetmap.us/)
+- Rust
+  - [Docs](https://www.rust-lang.org/learn) for different learning styles
+
```
Comment (Sparrow0hawk) [inline docs/developer_guide.md:32]:
```suggestion
  - [The Rust Book](https://doc.rust-lang.org/stable/book/)
```

File: docs/developer_guide.md [hunk 2]
```diff
@@ -32,11 +48,15 @@ If you're using Firefox locally to develop and get "import declarations may
 only appear at top level" errors, upgrade to at least Firefox 112, go to
 `about:config` and enable `dom.workers.modules.enabled`
 
+### Docker
+
 To run via Docker in VSCode you'll need:
+
 - [Docker](https://www.docker.com/)
 - [VSCode ](https://code.visualstudio.com/) with the extension ["Dev Containers"](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) installed
```
Comment (Sparrow0hawk) [inline docs/developer_guide.md:56]:
```suggestion
- [VSCode](https://code.visualstudio.com/) with the extension ["Dev Containers"](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) installed
```

trimming some whitespace in the link

**Comment by Pete-Y-CS** [comment]
I suppose we should mention that as yet we can't get nvm installed onto DfT laptops outside the office and therefore we can't get the right version of Node installed very easily. Also NPM doesn't work outside the office.

**Comment by Sparrow0hawk** [comment]
> I suppose we should mention that as yet we can't get nvm installed onto DfT laptops outside the office and therefore we can't get the right version of Node installed very easily. Also NPM doesn't work outside the office.

I think thats a separate internal issue which we'll document ourselves rather than include here. One for https://github.com/acteng/docs/blob/main/help/IT.md if you want to have a bash?

**Comment by Pete-Y-CS** [comment]
To be honest, barring IT issues, apart from Rust I've worked with these tools before, and I have done next to nothing with the Rust so far. I haven't found any particularly good resources to remind myself of these techs, mostly just looking at the official docs and so on.

**Review by Pete-Y-CS — APPROVED** [review]
To be honest, barring IT issues, I've not had much trouble getting going. Apart from Rust I've worked with these tools before, and I have done next to nothing with the Rust so far. I haven't found any particularly good resources to remind myself of these technologies, mostly just looking at the official docs and so on, so unfortunately not much to add.

LGTM!

**Review by robinlovelace-ate — COMMENTED** [review]
Woo will have to give this a go in a devcontainer, will let you know if the instructions work!

**Pull request merged**